src/utils/acquireSessionId.ts

```diff
@@ -0,0 +1,25 @@
+import { ulid } from 'ulid';
+
+export const SESSION_ID_KEY = 'yext_analytics_sessionId';
```

Comment:
This feels like a weird combination of camel case and snake case. Why not all camel case?

Reply:
ah, bad case of switching and copying this around.. I think chat-headless has its key for state in session storage as `yext_chat_conversation_state` so I will just follow that format and update this to `yext_analytics_session_id`